simple-git-hooks.js

```diff
@@ -296,14 +300,18 @@ function _getConfigFromFile(projectRootPath, fileName) {
 }
 
 /**
- * Validates the config, checks that every git hook is named correctly
+ * Validates the config, checks that every git hook or option is named correctly
  * @return {boolean}
- * @param {{string: string}} hooks
+ * @param {{string: string}} config
  */
-function _validateHooks(hooks) {
+function _validateHooks(config) {
+
+    for (let hookOrOption in config) {
```

Comment:
Should we use `Object.entries` or `Object.keys` here?

Reply:
I think this syntax looks cleaner, then one with `Object.keys`

Reply:
OK.